src/store/reducers/describe.ts

```diff
@@ -2,54 +2,81 @@ import {createSelector, Selector} from 'reselect';
 import {Reducer} from 'redux';
 
 import '../../services/api';
-import {TEvDescribeSchemeResult} from '../../types/api/schema';
 import {IConsumer} from '../../types/api/consumers';
-import {IDescribeRootStateSlice, IDescribeState} from '../../types/store/describe';
-import {IResponseError} from '../../types/api/error';
-import {createRequestActionTypes, createApiRequest, ApiRequestAction} from '../utils';
+import {
+    IDescribeRootStateSlice,
+    IDescribeState,
+    IDescribeAction,
+    IDescribeFetchAdditionalParams,
+} from '../../types/store/describe';
+import {createRequestActionTypes, createApiRequest} from '../utils';
 
-const FETCH_DESCRIBE = createRequestActionTypes('describe', 'FETCH_DESCRIBE');
+export const FETCH_DESCRIBE = createRequestActionTypes('describe', 'FETCH_DESCRIBE');
 
 const initialState = {
-    loading: false,
+    loading: true,
     wasLoaded: false,
     data: {},
+    currentDescribe: undefined,
+    currentDescribePath: undefined,
 };
 
-const describe: Reducer<
-    IDescribeState,
-    ApiRequestAction<typeof FETCH_DESCRIBE, TEvDescribeSchemeResult, IResponseError>
-> = (state = initialState, action) => {
+const describe: Reducer<IDescribeState, IDescribeAction> = (state = initialState, action) => {
     switch (action.type) {
         case FETCH_DESCRIBE.REQUEST: {
+            // Reset loading helps to keep correct loader render on tabs or path change
+            const resetLoading = action.additionalParams?.resetLoadingState;
```

Comment:
Let's share responsibility between different actions without additional params, which brings quite a complicated logic.
`REQUEST`, `RESET STATE`, `SET CURRENT DESCRIBE PATH` and so on.